src/kakarot/instructions/system_operations.cairo

```diff
@@ -551,9 +552,58 @@ namespace CreateHelper {
         pedersen_ptr: HashBuiltin*,
         range_check_ptr,
         bitwise_ptr: BitwiseBuiltin*,
-    }(sender_address: felt, nonce: felt) -> (evm_contract_address: felt) {
-        // TODO: see https://github.com/sayajin-labs/kakarot/issues/426
-        return (0xAbdE100700000000000000000000000000000000 + nonce,);
+    }(sender_address: felt, salt: felt) -> (evm_contract_address: felt) {
```

Comment:
I thought I would preserve our back and forth here about potential next steps re: the salt usage here.

> me: just to sanity check my understanding, in the case of create, the evm documentation refer to a nonce. in our codebase we have salt storage variable. 

> these terms are pretty much interchangeable?

> clement: actually I think that the nonce is sender related while our salt is currently global
> actually I think that the nonce is sender related while our salt is currently global

Reply:
yes, let's keep like that (`salt`)  and create an issue for investigating the usage of a true sender's nonce